documentation/jetty/modules/operations-guide/pages/jsf-taglibs/index.adoc

```diff
@@ -13,7 +13,7 @@
 
 = JavaServer Faces TagLibs
 
-If you want to use JSF with your webapp, you should copy the relevant jars from your implementation of choice into your `$JETTY_BASE` directory, ideally into `$JETTY_BASE/lib/ext`.
+If you want to use JSF with your web application, you should copy the relevant jars from your implementation of choice into your `$JETTY_BASE` directory, ideally into `$JETTY_BASE/lib/ext`.
```

Comment:
This is exactly the point I've been trying to make! This change is incorrect: as you want to use `web application` as a generic term, this sentence implies that you can use JSF with a core app or a static app, which is emphatically _not_ the case. To ensure the consistency of the nomenclature, we need to change every occurrence of `web application` where it _only_ applies to a Servlet Spec type application to read `ee web application`.  It would be much much simpler to just use `application` in the _new_  documentation of core and static apps. If there's no appetite to do that, then the documentation needs to be updated to read `ee web application` where applicable.

Reply:
I already did a good pass about specifying Jakarta EE web application, I missed this one, but it's obvious that the documentation is referring to a Jakarta EE technology (oh, look, the title is wrong).

I'll fix it.